verifiers/proofofsql/src/benchmarking.rs

```diff
@@ -0,0 +1,116 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::ProofOfSql;
+use frame_benchmarking::v2::*;
+use frame_system::RawOrigin;
+use hp_verifiers::Verifier;
+use pallet_verifiers::{VkOrHash, Vks};
+
+pub struct Pallet<T: Config>(crate::Pallet<T>);
+
+pub trait Config: crate::Config {}
+impl<T: crate::Config> Config for T {}
+pub type Call<T> = pallet_verifiers::Call<T, ProofOfSql<T>>;
+
+#[benchmarks(where T: pallet_verifiers::Config<ProofOfSql<T>>)]
+mod benchmarks {
+
+    use super::*;
+
+    #[benchmark]
+    fn submit_proof() {
```

Comment:
As a first iteration these benchmarks are ok and we can go in testnet with them: please open a Jira ticket to trace how we can do it better.

Reply:
I agree, we need to better understand the verifier computational complexity for doing proper benchmarks.